book/listings/todo_app/3/style.css

```diff
@@ -1,3 +1,3 @@
-.todo_row {
+box.todo_row {
```

Comment:
I'd use just `.todo_row` for this kind of thing (one less place to change if you decide to change the base widget from gtk::Box). But the tutorial also doesn't have any examples of free-standing style classes.

Reply:
Yes, the `box` doesn't add anything here.